cmake/modules/TmpCPack.cmake

```diff
@@ -0,0 +1,24 @@
+set(CPACK_PACKAGE_DIRECTORY "${CMAKE_BINARY_DIR}/_packages")
+set(CPACK_PACKAGE_VERSION ${PROJECT_VERSION})
+set(CPACK_PACKAGING_INSTALL_PREFIX /usr)
+
+set(CPACK_PACKAGE_CONTACT "Ignat Sergeev")
```

Comment:
- Are you sure you want to be the maintainer of this package instead of me?
- I suppose there also should be an email

Reply:
There definitely should be an email: https://gitlab.kitware.com/cmake/community/-/wikis/doc/cpack/PackageGenerators#debian-generator-specific-settings